### PR TITLE
New version: Jubler.App version 8.0.0

### PR DESCRIPTION
**Pull request by teras**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133810)

### CODE REVIEW

File: manifests/j/Jubler/App/8.0.0/Jubler.App.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Jubler.App
+PackageVersion: 8.0.0
+InstallerType: inno
+ReleaseDate: 2024-01-08
+Installers:
+- Architecture: x86
```
Comment (Exorcism0666) [inline manifests/j/Jubler/App/8.0.0/Jubler.App.installer.yaml:9]:
```suggestion
- Architecture: x64
```

File: manifests/j/Jubler/App/8.0.0/Jubler.App.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Jubler.App
+PackageVersion: 8.0.0
+InstallerType: inno
+ReleaseDate: 2024-01-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/teras/Jubler/releases/download/v8.0.0/Jubler-8.0.0.x64.exe
+  InstallerSha256: B67B7EB6AD94F3367F76C05CDC8AAE08185F022C2F616B094719EC8969E75933
```
Comment (Exorcism0666) [inline manifests/j/Jubler/App/8.0.0/Jubler.App.installer.yaml:11]:
```suggestion
  InstallerSha256: B67B7EB6AD94F3367F76C05CDC8AAE08185F022C2F616B094719EC8969E75933
- Architexture: x86
  InstallerUrl: https://github.com/teras/Jubler/releases/download/v8.0.0/Jubler-8.0.0.x32.exe
  InstallerSha256: f3f4ebb933fca74fefd044ea4456db2a413026d0d19978ddf99ec1e7bfd6aa7f
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-12T18:12:41.0836054Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][1] Description: Missing required property 'Architecture'. Error context: <root>[Installers] Description: Failed to validate item #1 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: Jubler.App.installer.yaml Manifest Warning: Unknown field. [Architexture] Manifest Error: Invalid field value. [Architecture]

(Automated response - build 563.)

File: manifests/j/Jubler/App/8.0.0/Jubler.App.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: Jubler.App
+PackageVersion: 8.0.0
+InstallerType: inno
+ReleaseDate: 2024-01-08
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/teras/Jubler/releases/download/v8.0.0/Jubler-8.0.0.x64.exe
+  InstallerSha256: B67B7EB6AD94F3367F76C05CDC8AAE08185F022C2F616B094719EC8969E75933
+- Architexture: x86
```
Comment (stephengillie) [inline manifests/j/Jubler/App/8.0.0/Jubler.App.installer.yaml:12]:
```suggestion
- Architecture: x86
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**